internal/lsp/completions/providers/builtins.go

```diff
@@ -29,6 +29,11 @@ func (*BuiltIns) Run(c *cache.Cache, params types.CompletionParams, _ *Options)
 		return []types.CompletionItem{}, nil
 	}
 
+	// default rules cannot contain calls
+	if strings.HasPrefix(strings.TrimSpace(currentLine), "default") {
```

Comment:
```suggestion
	if strings.HasPrefix(strings.TrimSpace(currentLine), "default ") {
```
I could call my rule `defaultrule` and I'd be surprised why there are no builtin completions...? Or am I misreading  this? 😅

Reply:
I think that makes sense yeah!